strabo/strabo/templates/public/map.html

```diff
@@ -100,10 +100,17 @@ <h3 id='section_num' style="padding-top: 30px;display: inline;font-size: 1.4em;"
 {% block footer %}
     {{ super() }}
     <script src="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
-    <!-- Load in geojson objects with janky cache-breaking solution so that changes to 
-    js file will be reflected on map -->
-    <script>document.write('<script src="/../static/js/{{ INTPT_FILE }}?dev=' + Math.floor(Math.random() * 100) + '"\><\/script>');</script>
+    <script>
```

Comment:
Would be nice to have to this in a microfile at some point, but okay here for now